source/ed/build_helper.h

```diff
@@ -93,7 +94,8 @@ struct VJ {
        const std::string& physical_mode = "",
        const uint32_t start_time = 0,
        const uint32_t end_time = 0,
-       const uint32_t headway_secs = 0);
+       const uint32_t headway_secs = 0,
+       const bool bike_accepted = true);
```

Comment:
you can remove it from the constructor, since we prefer to use the chained method